UnityProject/Assets/Scripts/UI/UI_HeartMonitor.cs

```diff
@@ -118,15 +118,17 @@ private void CheckHealth()
 		if (overallHealthCache <= 30 &&
 			overallHealthCache > 0)
 		{
-			SoundManager.Stop("Critstate");
+			// JESTER
+			//SoundManager.Stop("Critstate");
 			CurrentSpriteSet = 4;
 			pulseImg.sprite = StatesSprites[4].SP[currentSprite];
 			overlayCrits.SetState(OverlayState.injured);
 		}
 		if (overallHealthCache <= 0 &&
 			overallHealthCache < 15)
 		{
-			SoundManager.Play("Critstate");
+			// JESTER
+			//SoundManager.Play("Critstate");
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)